requirements.txt

```diff
@@ -50,7 +50,7 @@ beautifulsoup4
 lxml
 
 # 3.141 was missing socksVersion, 3.150 was not in pypi, so we try 4.1.0
```

Comment:
I would just delete this comment TBH

Reply:
yeah, it doesn't make any sense right now